x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/upgrade_flyout_subheader.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiText } from '@elastic/eui';
+import { camelCase, startCase } from 'lodash';
+import type { FieldsDiff } from '../../../../../../common/api/detection_engine';
+import { FormattedDate } from '../../../../../common/components/formatted_date';
+import { SeverityBadge } from '../../../../../common/components/severity_badge';
+import { ModifiedBadge } from '../../../../rule_management/components/rule_details/three_way_diff/badges/modified_badge';
+import type { RuleUpgradeState } from '../../../../rule_management/model/prebuilt_rule_upgrade';
+import { fieldToDisplayNameMap } from '../../../../rule_management/components/rule_details/diff_components/translations';
+import * as i18n from './translations';
+
+interface UpgradeFlyoutSubHeaderProps {
+  ruleUpgradeState: RuleUpgradeState;
+}
+
+export const UpgradeFlyoutSubHeader = memo(function UpgradeFlyoutSubHeader({
+  ruleUpgradeState,
+}: UpgradeFlyoutSubHeaderProps): JSX.Element {
+  const lastUpdate = (
+    <EuiText size="s">
+      <strong>
+        {i18n.LAST_UPDATE}
+        {':'}
+      </strong>{' '}
+      {i18n.UPDATED_BY_AND_WHEN(
+        ruleUpgradeState.current_rule.updated_by,
+        <FormattedDate value={ruleUpgradeState.current_rule.updated_at} fieldName="" />
+      )}
+    </EuiText>
+  );
+
+  const severity = (
+    <EuiFlexGroup gutterSize="xs">
+      <EuiText size="s">
+        <strong>
+          {i18n.SEVERITY}
+          {':'}
+        </strong>
+      </EuiText>
+      <SeverityBadge value={ruleUpgradeState.target_rule.severity} />
+    </EuiFlexGroup>
+  );
+
+  const customized = ruleUpgradeState.current_rule.rule_source.type === 'external' &&
+    ruleUpgradeState.current_rule.rule_source.is_customized && (
+      <ModifiedBadge tooltip={i18n.RULE_MODIFIED_BADGE_DESCRIPTION} />
+    );
+
+  const fieldsDiff: FieldsDiff<Record<string, unknown>> = ruleUpgradeState.diff.fields;
+  const fieldsNamesWithUpdates = Object.keys(ruleUpgradeState.fieldsUpgradeState).filter(
+    (fieldName) => fieldsDiff[fieldName].has_update
+  );
+  const fieldUpdates = fieldsNamesWithUpdates.length > 0 && (
+    <EuiText size="s">
+      <strong>
+        {i18n.FIELD_UPDATES}
+        {':'}{' '}
+        {fieldsNamesWithUpdates
```

Comment:
I think field names need to be moved out of `<strong>` to match the design.

**Current**
<img width="645" alt="Scherm­afbeelding 2024-12-05 om 17 23 37" src="https://github.com/user-attachments/assets/7e90541f-9f7c-4021-b641-ccbb2438853d">

**Design**
<img width="591" alt="Scherm­afbeelding 2024-12-05 om 17 22 42" src="https://github.com/user-attachments/assets/8e879c51-3c38-4489-899e-9853e87ce062">